projects/kit/src/lib/masks/time/processors/max-validation-preprocessor.ts

```diff
@@ -3,13 +3,13 @@ import type {MaskitoPreprocessor} from '@maskito/core';
 import {TIME_FIXED_CHARACTERS} from '../../../constants';
 import type {MaskitoTimeMode, MaskitoTimeSegments} from '../../../types';
 import {escapeRegExp} from '../../../utils';
-import {padTimeSegments, validateTimeString} from '../../../utils/time';
+import {padEndTimeSegments, validateTimeString} from '../../../utils/time';
 
 export function createMaxValidationPreprocessor(
     timeSegmentMaxValues: MaskitoTimeSegments<number>,
     timeMode: MaskitoTimeMode,
 ): MaskitoPreprocessor {
-    const paddedMaxValues = padTimeSegments(timeSegmentMaxValues);
+    const paddedMaxValues = padEndTimeSegments(timeSegmentMaxValues);
```

Comment:
If I've not mistaken, `padStartTimeSegments` is more suitable here 🤔 

Probably, case with single digit was always broken :D

https://stackblitz.com/edit/time-segment-max-value-bug-single-digit?file=mask.ts

```ts
const timeOptions = maskitoTimeOptionsGenerator({
    mode: 'HH:MM',
    timeSegmentMaxValues: {hours: 4}, // you can enter 09:09 :(
});
```

https://stackblitz.com/edit/time-segment-max-value-bug-single-digit?file=mask.ts

Reply:
```suggestion
    const paddedMaxValues = padStartTimeSegments(timeSegmentMaxValues);
```

\+ add some cypress tests for single-digit cases, please

Reply:
Updated. Unfortunately, Cypress is no longer with me.

Reply:
> Updated. Unfortunately, Cypress is no longer with me.

I'll do it later in separate PR 👌